idact/allocate_resources.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+"""Console script for idact-deploy-notebook.
+
+How to run::
+
+  python -m idact.allocate_resources --help
+
+.. click:: idact.allocate_resources:main
+
+"""
+
+import sys
+from idact.detail.allocate_resources_app.main import main
+
+if __name__ == "__main__":
+    sys.exit(
```

Comment:
Why this is split into two lines ?

Reply:
Because the comment is too long for the line, it must stay to disable pylint check